PA_21.py

```diff
@@ -0,0 +1,27 @@
+#LC 256 - Paint House
+#Time Complexity - O(n) Not sure, can you comment?

```

Comment:
Since you are using recursion, the time complexity will be exponential
In this case, cost power color

Reply:
Try the iterative solution for this as well.